asset.py

```diff
@@ -31,3 +32,5 @@ def __post_init__(self):
         if type(self.publication_date) and not self.publication_date:
             print(f"{type(self.publication_date)=}\n{self.publication_date=}")
             self.publication_date = None
+        pattern = "【(.+?)】"
```

Comment:
```suggestion
        pattern = "【.+?】"
```

正規表現の `()` が何を意味するか調べておいてください。

Reply:
さらに言うと、こう言う「除外したいパターン」は今後も変わる可能性があるから、外部化しておく方がベター。

Reply:
承知しました。
確かに、こういう文字列の処理はまとめて外部に移動させたいですね。
`【】`以外にもたくさん出てきそうですもんね。